save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationContestsMenu.kt

```diff
@@ -89,8 +94,6 @@ private val contestsTable: FC<OrganizationContestsTableProps<ContestDto>> = tabl
             }
         }
     },
-    initialPageSize = 10,
```

Comment:
?

Reply:
There is no need to pass `initialPageSize = 10` because the default value for this property is 10 and is contained in the `INITIAL_TABLE_PAGE_SIZE` constant.